<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AlphaCopy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 48, 59);
color:rgb(220, 220, 220);
border-radius: 10px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="initPage">
       <widget class="QLabel" name="appTitle">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>421</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Laksaman</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(33, 188, 162);</string>
        </property>
        <property name="text">
         <string>&lt;strong&gt;Alpha&lt;/strong&gt;Copy</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="logoLabel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>191</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="loadingDevicesLb">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>421</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(33, 188, 162);</string>
        </property>
        <property name="text">
         <string>&lt;strong&gt;LOADING&lt;/strong&gt; DEVICES...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="openSourceLb">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>421</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Laksaman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(143, 66, 231);</string>
        </property>
        <property name="text">
         <string>&lt;strong&gt;OPEN SOURCE&lt;/strong&gt; PROJECT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QProgressBar" name="devicesProgB">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>200</y>
          <width>361</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
	background-color: rgb(100, 130, 200);
	color: rgb(230, 230, 230);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0138249 rgba(0, 0, 0, 255), stop:0.0184332 rgba(143, 66, 231, 255), stop:0.976959 rgba(14, 218, 195, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="devNotFound">
       <widget class="QLabel" name="notFoundLb">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>401</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 4);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;DEVICES NOT FOUND!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="scanAgainPB">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>251</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0138249 rgba(0, 0, 0, 255), stop:0.0184332 rgba(143, 66, 231, 255), stop:0.976959 rgba(14, 218, 195, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}
QPushButton::pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(222, 151, 255, 255), stop:1 rgba(54, 237, 216, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}</string>
        </property>
        <property name="text">
         <string>Scan Again?</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="foundDevs">
       <widget class="QLabel" name="foundDevicesLb">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(35, 175, 136);</string>
        </property>
        <property name="text">
         <string>&lt;strong&gt;FOUND DEVICES</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="arrowLb">
        <property name="geometry">
         <rect>
          <x>182</x>
          <y>80</y>
          <width>79</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(35, 175, 136);</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="sdLE">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	
	color: rgb(200, 200, 200);
	background-color: rgb(46, 48, 69);

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>SD Label</string>
        </property>
       </widget>
       <widget class="QPushButton" name="makeCopyPB">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>251</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0138249 rgba(0, 0, 0, 255), stop:0.0184332 rgba(143, 66, 231, 255), stop:0.976959 rgba(14, 218, 195, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}
QPushButton::pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(222, 151, 255, 255), stop:1 rgba(54, 237, 216, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}</string>
        </property>
        <property name="text">
         <string>Make Copy</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="hddLE">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>130</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	
	color: rgb(200, 200, 200);
	background-color: rgb(46, 48, 69);

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>HDD Label</string>
        </property>
       </widget>
       <widget class="QLabel" name="sdIconLb">
        <property name="geometry">
         <rect>
          <x>63</x>
          <y>60</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="hddIconLb">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>91</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="progBar">
       <widget class="QLabel" name="copyingFileLb">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>401</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(35, 175, 136);</string>
        </property>
        <property name="text">
         <string>&lt;strong&gt;COPYING FILES...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QProgressBar" name="copyProgrB">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>381</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
	background-color: rgb(100, 130, 200);
	color: rgb(230, 230, 230);
	border-style: none;
	border-radius: 35px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 35px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0138249 rgba(0, 0, 0, 255), stop:0.0184332 rgba(143, 66, 231, 255), stop:0.976959 rgba(14, 218, 195, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%p files</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="successCopy">
       <widget class="QLabel" name="SuccesfulLb">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>401</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(35, 175, 136);</string>
        </property>
        <property name="text">
         <string>&lt;STRONG&gt;SUCCESFUL COPY</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="DonePB">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>251</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0138249 rgba(0, 0, 0, 255), stop:0.0184332 rgba(143, 66, 231, 255), stop:0.976959 rgba(14, 218, 195, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}
QPushButton::pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(222, 151, 255, 255), stop:1 rgba(54, 237, 216, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}</string>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="errorsFound">
       <widget class="QPushButton" name="errorDonePB">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>251</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0138249 rgba(0, 0, 0, 255), stop:0.0184332 rgba(143, 66, 231, 255), stop:0.976959 rgba(14, 218, 195, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(255, 0, 4);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}
QPushButton::pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(222, 151, 255, 255), stop:1 rgba(54, 237, 216, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}</string>
        </property>
        <property name="text">
         <string>Done!</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
       <widget class="QLabel" name="errorsFoundLb">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>401</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 4);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ERRORS FOUND!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ejectDevs">
       <widget class="QPushButton" name="ejectPB">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>251</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0138249 rgba(0, 0, 0, 255), stop:0.0184332 rgba(143, 66, 231, 255), stop:0.976959 rgba(14, 218, 195, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}
QPushButton::pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(222, 151, 255, 255), stop:1 rgba(54, 237, 216, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}</string>
        </property>
        <property name="text">
         <string>Eject Devices</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="newCopy">
       <widget class="QPushButton" name="newCoyPB">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>251</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.0138249 rgba(0, 0, 0, 255), stop:0.0184332 rgba(143, 66, 231, 255), stop:0.976959 rgba(14, 218, 195, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}
QPushButton::pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(222, 151, 255, 255), stop:1 rgba(54, 237, 216, 255));
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 30px;
	text-align: center;
}</string>
        </property>
        <property name="text">
         <string>Make New Copy</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
       <widget class="QLabel" name="ejectedLb">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>421</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(35, 175, 136);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;EJECTED DEVICES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>scanAgainPB</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>scan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>makeCopyPB</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ejectPB</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>return_to_main_page()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DonePB</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scan()</slot>
  <slot>copy()</slot>
  <slot>done()</slot>
  <slot>return_to_main_page()</slot>
 </slots>
</ui>
